config.yml

```diff
@@ -1,59 +1,18 @@
-experiment_name: my_example
-n_repeats: 5
+experiment_name: debug_config
+n_repeats: 1
```

Comment:
OK, sorry it is unclear: try not to play with `config.yml` with has to stay a stable default config file for a basic experimentation. You can use `config_quick_debug.yml`, although it is also constrained by being simple enough so that the automated CI tests don't take too long. Actually we should clarify all that with a `default_config.yml`, a `ci_tests_config.yml` and a `dowhatyouwant_config.yml`